client/docs/websites.md

```diff
@@ -0,0 +1,34 @@
+# Monorepo: Websites
+
+This monorepo has special support for website packages, in the `web/` folder. These are quite a bit more complex than module packages. While technically there is no specific restriction on what can be used to build or develop a website package, it is recommended that you use [Snowpack](https://www.snowpack.dev/). This doc. will assume that you're using Snowpack and that you started with the `templates/website-template` configuration.
```

Comment:
What possible reasons are there for using something other than Snowpack? Would it make sense to explain why a divergence would occur? In general it seems any standard new package should use Snowpack, so we should recommend it in stronger terms here.

Reply:
This goes back to me "not wanting to hide how the repo works", but I can think of a few reasons. If you were using a static site generator, say for a development blog, you wouldn't want to demand Snowpack because it's not needed. All that is technically needed is a `dev` and `build` command. The simplicity of that requirement makes me feel uneasy about demanding the usage of Snowpack.